### ISSUE TITLE
Add "space setup" practices

### ISSUE THREAD

**Issue by rdebeasi** [openpracticelibrary/openpracticelibrary#98]
To cover the logistics of setting up a physical space that invites collaboration and creative thinking.

We have a great [internal doc](https://docs.google.com/document/d/1M0OWc_-A8L817tKmEnbt4gF4K85tycuo50Zo9ft_qyo/edit) - let's make it public! @valyonchev and others, feel free to submit a PR so that you get credit for this in the GitHub history. :)

**Comment by valyonchev** [comment]
On it. I have already updated our draft documents last week. I will find time to raise a PR this week.

**Comment by mwalker5000** [comment]
I found this issue -- I think https://github.com/openpracticelibrary/openpracticelibrary/issues/104 is a duplicate so I'm closing that one and adding my comments here.

I think we'll need to look both at the doc Ryan referenced earlier, as well as the one I've listed below (which was basically a cleaned up version of that doc that has had additional contributions from others since then). 

Original comments:

Include info on best practices and lessons learned from our 2+ years building and renting facilities to create Labs spaces.

Include:

* excerpts from design inspiration decks used from prior labs buildouts
* pictures of labs office schematics
* pictures of actual labs (pop-up, permanent, etc)
* writeup of what we've learned
* Links to additional pages on materials that we like to use

Within RHT [there is a document that exists as a placeholder for this stuff].(https://docs.google.com/document/d/1A2vms6HqSblbAyXw8qn1Lpjeo3CCaU_i_-L5FJzzoDg/edit#heading=h.xzz54n8rlx4c)
